<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workField</class>
 <widget class="QDialog" name="workField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>669</width>
     <height>505</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="reciversListGroupBoxChecked">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>307</y>
      <width>659</width>
      <height>157</height>
     </rect>
    </property>
    <property name="title">
     <string>Печатать список рассылки</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Экземпляр №1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="reciverOne_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Экземпляр №2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="reciverTwo_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Экземпляр №3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Экземпляр №4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Экземпляр №5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="reciverFour_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="reciverThree_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="reciverFive_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="commonGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>88</y>
      <width>261</width>
      <height>213</height>
     </rect>
    </property>
    <property name="title">
     <string>Общие атрибуты</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Гриф</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QComboBox" name="secretCBox">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Номер экземпляра</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="3">
      <widget class="QLineEdit" name="copyNumberLineEd"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пункт</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QLineEdit" name="punktLineEd"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Номер МБ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="4">
      <widget class="QLineEdit" name="mbNumberLineEd"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Количество листов</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLineEdit" name="pageCountEd">
       <property name="inputMask">
        <string>Dddd; </string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="tmplLabel">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Общие шаблоны</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QToolButton" name="editTemplatesTBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Редактирование текщего шаблона</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/handbook.png</normaloff>:/images/handbook.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QToolButton" name="addTemplatesTBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/edit_add.png</normaloff>:/images/edit_add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QToolButton" name="dellTemplatesTBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/edit_remove.png</normaloff>:/images/edit_remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QComboBox" name="templatesCbox">
       <property name="toolTip">
        <string>Выберите шаблон,который будет примен к текущему документу</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QToolButton" name="switch_Local_GlolbalBtn">
       <property name="toolTip">
        <string>Выбор общих или индивидуальных шаблонов</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/user-32.png</normaloff>:/images/user-32.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="titleGroupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>659</width>
      <height>77</height>
     </rect>
    </property>
    <property name="title">
     <string>Название документа...</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="docName_plainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>641</width>
         <height>41</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="cancelBtn">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>476</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="paperAccountsOutSide">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>479</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Листы учтены в секретном органе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previewBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>476</y>
      <width>285</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Печать учетной метки на обратной стороне листа</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/win/fileprint.png</normaloff>:/images/win/fileprint.png</iconset>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="lastPageGroupBoxChecked">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>88</y>
      <width>391</width>
      <height>213</height>
     </rect>
    </property>
    <property name="title">
     <string>Штамп последней страницы</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Исполнитель</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="executor_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отпечатал</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pressman_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Телефон</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Инв. №</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="invNumber_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="telephone_lineEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="dateField_dateEd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1752</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>520</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="AnsLabel">
      <property name="text">
       <string>Статус:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lastPageGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>executor_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastPageGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>pressman_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastPageGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>telephone_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastPageGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>invNumber_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastPageGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateField_dateEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reciversListGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>reciverOne_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reciversListGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>reciverTwo_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reciversListGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>reciverThree_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reciversListGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>reciverFour_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reciversListGroupBoxChecked</sender>
   <signal>toggled(bool)</signal>
   <receiver>reciverFive_lineEd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>workField</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
